apix_documentation_swagger_v4.yaml

```diff
@@ -0,0 +1,1186 @@
+        This endpoint is used to get all your attempted migrations.
+      operationId: getMigrations
+      parameters:
+        - name: page
+          in: query
+          description: It allows you to display any page of the migrations’ list
+          required: false
+          schema:
+            type: integer
+      responses:
+        200:
+          description: successful response
+          content:
+            application/vnd.regalii.v4.0+json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/Migration'
+                allOf:
+                  - $ref: '#/components/schemas/MetaData'
+                  - $ref: '#/components/schemas/NavigationLinks'
+
+                example:
+                  data:
+                    - id: b4aa7d78-6f16-4ddd-887d-9f5c0b730a26
+                      type: migration
+                      attributes:
+                        status: failed
+                        requested_at: '2018-06-15 14:16:16 UTC'
+                        migrated_at: '2018-06-15 16:34:54 UTC'
+                        error:
+                          code: InvalidCredentials
+                          detail: Invalid login or password
+                  # TODO: Include examples of different statuses and error codes
+                  meta:
+                    total: 23
+                    total-pages: 1
+                  links:
+                    self: "http://localhost:3000/migrations?page=1"
+                    next: null
+                    prev: null
+
+  /migrations/{id}:
+    get:
+      tags:
+        - xChange
+      summary: Gets the last migration attempt of a bill
+      description: >-
+        This endpoint gets the last migration attempt of a bill.
+      operationId: getMigration
+      parameters:
+        - name: id
+          in: path
+          description: ID of the bill whose migration information you wish to
+            obtain
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: successful response
+          content:
+            application/vnd.regalii.v4.0+json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    allOf:
+                      - $ref: '#/components/schemas/Migration'
+
+                example:
+                  data:
+                    id: b4aa7d78-6f16-4ddd-887d-9f5c0b730a26
+                    type: migration
+                    attributes:
+                      status: failed
+                      requested_at: '2018-06-15 14:16:16 UTC'
+                      migrated_at: '2018-06-15 16:34:54 UTC'
+                      error:
+                        code: InvalidCredentials
+                        detail: Invalid login or password
+
```

Comment:
Missing docs for `POST /migration body: { id: {uuid} }`

Reply:
https://arcusfi.atlassian.net/browse/USX-881